common/OpTestHMC.py

```diff
@@ -927,20 +927,20 @@ def is_perfcollection_enabled(self):
         if  rc:
             return True
 	return False
-    
```

Comment:
Do we need this additional commit? It changes the code(cosmetic changes) added by previous commit? Can we fold this into previous one?

Reply:
as i was looking to add changes align with logging as changes , let me see if some thing messed up while , let me resend

Reply:
yeah.. you can just ammend the changes to same previous commit

Reply:
yes closing this PR as have logic separation of commit as some while addressing review comment `bank line` goes as diffrent commit